test/test-find-visualstudio.js

```diff
@@ -171,7 +189,7 @@ describe('find-visualstudio', function () {
     finder.parseData(null, '[]', '', (info) => {
       assert.ok(/find .* Visual Studio/i.test(finder.errorLog[0]), 'expect error')
       assert.ok(!info, 'no data')
-    })
+    }, { checkIsArray: true })
```

Comment:
Input parameters order is bad, the callback will be passed as options. Also, seems this test is outdated as the callback is never called in general (no `parseData` parameter for any type of callback).